src/ConfigUpdaterResult.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace Drupal\commerce;
+
+/**
+ * Result object for displaying configuration update results.
+ */
+class ConfigUpdaterResult implements ConfigUpdaterResultInterface {
+
+  /**
+   * Array of configuration names that failed to update.
```

Comment:
"Array of" is redundant, and something that should be told to us by @var. Just say "The configuration names that failed to update."